src/UI/Art/SpriteArtist/PlayerSetupCommanderArtist.java

```diff
@@ -0,0 +1,178 @@
+package UI.Art.SpriteArtist;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import CommandingOfficers.CommanderInfo;
+import Engine.IController;
+import Engine.OptionSelector;
+import UI.InputHandler.InputAction;
+import UI.PlayerSetupCommanderController;
+import UI.UIUtils;
+
+public class PlayerSetupCommanderArtist
+{
+  private static HashMap<Integer, CommanderPanel> coPanels = new HashMap<Integer, CommanderPanel>();
+
+  public static void draw(Graphics g, IController controller, ArrayList<CommanderInfo> infos, Color playerColor)
+  {
+    PlayerSetupCommanderController control = (PlayerSetupCommanderController)controller;
+    if( null == control )
+    {
+      System.out.println("WARNING! PlayerSetupCommanderController was given the wrong controller!");
+    }
+
+    // Define the draw space
+    int drawScale = SpriteOptions.getDrawScale();
+    Dimension dimensions = SpriteOptions.getScreenDimensions();
+    int myWidth = dimensions.width / drawScale;
+    int myHeight = dimensions.height / drawScale;
+    BufferedImage image = SpriteLibrary.createTransparentSprite(myWidth, myHeight);
+    Graphics myG = image.getGraphics();
+
+    // Allocate space for the Commander faces.
+    int cursorSizePx = SpriteLibrary.getCursorSprites().getFrame(0).getWidth(); // Cursor frames are square.
+    int coZoneYCenter = myHeight / 2;
+
+    /////////////// Tooltip ////////////////////////////
+    BufferedImage tooltip = SpriteUIUtils.makeTextFrame("Press Q for more info", 3, 2);
+    myG.drawImage(tooltip, myWidth - tooltip.getWidth(), 3, null);
+
+    /////////////// Commander Portrait //////////////////////
+    int highlightedCommander = control.getSelectedCommander();
+    BufferedImage likeness = SpriteLibrary.getCommanderSprites( infos.get(highlightedCommander).name ).body;
+    myG.drawImage(likeness, myWidth-likeness.getWidth(), myHeight-likeness.getHeight(), null);
+
+    /////////////// Team Panels //////////////////////
+    // Calculate the vertical space each player panel will consume.
+    int panelBuffer = 3;
+    int panelHeight = CommanderPanel.PANEL_HEIGHT+panelBuffer;
+
+    // If we are moving from one option to another, calculate the intermediate draw location.
+//    if( animHighlightedPlayer != highlightedPlayer )
+//    {
+//      double slide = SpriteUIUtils.calculateSlideAmount(animHighlightedPlayer, highlightedPlayer);
+//      animHighlightedPlayer += slide;
+//    }
```

Comment:
No slide?

Reply:
I definitely intend to re-add sliding. Here and everywhere.